key.core/src/main/java/de/uka/ilkd/key/logic/Term.java

```diff
@@ -43,8 +43,11 @@
  * supported: {@link Term#execPostOrder(Visitor)} and {@link Term#execPreOrder(Visitor)}.
  */
 public interface Term
-        extends SVSubstitute, Sorted, EqualsModProofIrrelevancy, org.key_project.logic.Term {
-    @Override
+        extends SVSubstitute, Sorted, TermEqualsModProperty, org.key_project.logic.Term {
+
+    /**
+     * The top operator (e.g., in "A and B" this is "and", in f(x,y) it is "f").
+     */
```

Comment:
Why is this no longer an override?

Reply:
That is indeed strange. I took another look at the first merge I did in IntelliJ and it added the correct import and overrides at the other positions but not there. I will add the override and remove the comment.

Reply:
Somehow that broke the Unit tests. Does this also happen on your machine?

Reply:
Tests are still running on my machine, I will reply again when they are finished.

Reply:
I ran testFast and the only tests failing as per usual were the ones concerned with symbolic execution.
![grafik](https://github.com/KeYProject/key/assets/93993502/1473bb02-10ef-4950-b881-077a407ecc55)

Reply:
Running test also only fails for symbolic execution.
![grafik](https://github.com/KeYProject/key/assets/93993502/3667d6e6-22aa-46eb-b84b-981d2af31f7d)

Reply:
I was just checking this this morning, but it strangely did not fail locally.
![grafik](https://github.com/KeYProject/key/assets/93993502/78f89810-5910-4f35-b421-a9009dcf34cb)

(And just to be absolutely sure I synced my fork and tried it again... Still passes.)

Reply:
That makes some sense. Then there must be a bug outside of the KeY code. Either CI or even a Java bug. Especially since the Windows tests pass.

Reply:
The unit tests are now passing but testRunAllFunProofs fails. This also happens locally and even on the main branch.

Reply:
Running the test locally results in the following problem:
![grafik](https://github.com/KeYProject/key/assets/93993502/8ac098c5-ef4c-47b2-9ae3-354121829dad)
Followed by:
![grafik](https://github.com/KeYProject/key/assets/93993502/85c4fa06-5372-4f10-874f-94a58c2ccc80)

Reply:
The tests work on my main branch

Reply:
The tests work on my local main branch

Reply:
After syncing the fork, we are now back at only the unit tests failing. Might this be solved clearing the cache again?

Reply:
Probably, yes. I just cleared some caches and I am gonna restart it in a bit. Thanks for checking!